### PR TITLE
Jetty Runs and Test do there job.

### PR DESCRIPTION
**Pull request by ChielZelders**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -35,7 +56,24 @@ dependencies {
     testCompile group: 'org.hamcrest', name: 'hamcrest-library', version: '1.3'
 
     compile 'org.glassfish.jersey.containers:jersey-container-servlet:2.14'
+    compile "com.sun.jersey:jersey-json:1.9.1"
     compile 'org.jsoup:jsoup:1.8.1'
     compile 'org.jfrog.buildinfo:build-info-extractor-gradle:3.1.1'
     compile group: 'com.owlike', name: 'genson', version: '0.94'
+    integrationTestCompile 'org.assertj:assertj-core:3.0.0'
+    compile group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.5.4'
```
Comment (preslavgerchev) [inline build.gradle]: Do we need `org.apache.httpcomponents` for the main code or only for test? If so it shouldn't it be `testCompile`?

File: build.gradle [hunk 2]
```diff
@@ -35,7 +56,24 @@ dependencies {
...
+    compile group: 'org.apache.httpcomponents', name: 'httpcore', version: '4.4.8'
```
Comment (preslavgerchev) [inline build.gradle]: Same as above

File: src/test/java/ConversionUtil.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import javax.swing.text.html.parser.Entity;
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class ConversionUtil {
+    private Genson genson;
+    static private ConversionUtil instance;
+    static public ConversionUtil getConversionUtil(){if(instance == null){instance = new ConversionUtil();} return instance;}
+    public ConversionUtil(){
```
Comment (preslavgerchev) [inline src/test/java/ConversionUtil.java]: You should make the constructor `private`. Else it defeats the purpose of a singleton as I can now do `new ConversionUtil()` and get a new instance.

File: build.gradle (hunk 1 above)
Comment (ChielZelders) [reply build.gradle]: yes you are right!

**Review by Critical187 — APPROVED** [review]
hey thats mildly decent

**Pull request merged**